hw/top_earlgrey/syn/asic.constraints.sdc

```diff
@@ -162,6 +162,16 @@ set SPI_DEV_TCK 16.0
 create_clock -name SPI_DEV_CLK  -period ${SPI_DEV_TCK} [get_ports ${SPI_DEV_CLK_PIN}]
 set_clock_uncertainty ${SETUP_CLOCK_UNCERTAINTY} [get_clocks SPI_DEV_CLK]
 
+create_generated_clock -name SPI_DEV_IN_CLK -source SPI_DEV_CLK -divide_by 1 \
+    [get_pins top_earlgrey/u_spi_device/u_clk_spi_in_buf/clk_o]
+create_generated_clock -name SPI_DEV_OUT_CLK -source SPI_DEV_CLK -divide_by 1 \
```

Comment:
ah did this actually work? would be great if it did!

Reply:
This should work, I think I used something like this -divide_by 1 previously to get phase-offset output of PLLs.  (I'm not sure of the interplay with -invert though)